.github/workflows/CI.yml

```diff
@@ -38,8 +38,8 @@ jobs:
       matrix:
         # for core tests (latest and oldest supported versions)
         version:
-          - '1.7'  # oldest
-          - '1'    # latest
+          - '1.10' # oldest
+        #  - '1'    # latest
```

Comment:
We have just temporary commented, right?

Reply:
yes, because the latest julia version is also `v1.10`.
But the julia version `v1.11` is coming soon, we can uncomment this part after that.